apps/judicial-system/backend/migrations/20221020110402-update-notification.js

```diff
@@ -0,0 +1,118 @@
+'use strict'
+
+// import { QueryInterface, Sequelize } from 'sequelize'
+
+module.exports = {
+  // async up(queryInterface: QueryInterface, Sequelize: Sequelize) {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.transaction(async (transaction) =>
+      Promise.all([
+        queryInterface
+          .addColumn(
+            'notification',
+            'recipients_json',
+            {
+              defaultValue: [],
+              type: Sequelize.ARRAY(Sequelize.JSON),
+              allowNull: false,
+            },
+            { transaction },
+          )
+          .catch((err) => {
+            console.error(
+              'Error adding column recipients_json to table notification',
+              err,
+            )
+            throw err
+          }),
+        queryInterface.sequelize
```

Comment:
It would be safer to chain the database updates with `then` to make sure they are executed in the right order.